### PR TITLE
Convert axis to fromToMotion()

### PR DESCRIPTION
**Pull request by alantreadway**

### CODE REVIEW

File: packages/ag-charts-community/src/chart/series/cartesian/__image_snapshots__/bar-series-test-ts-bar-series-add-animation-for-column-time-x-axis-number-y-axis-should-animate-at-50-1-snap.png
Comment (lsjroberts) [inline packages/ag-charts-community/src/chart/series/cartesian/__image_snapshots__/bar-series-test-ts-bar-series-add-animation-for-column-time-x-axis-number-y-axis-should-animate-at-50-1-snap.png:1]: Just checking the reason the bottom axis labels aren't appearing at this point is due to the phases, not some other issue?

**Review by lsjroberts — APPROVED** [review]
LGTM

**Review by alexanderby — APPROVED** [review]

File: packages/ag-charts-community/src/chart/series/cartesian/__image_snapshots__/bar-series-test-ts-bar-series-add-animation-for-column-time-x-axis-number-y-axis-should-animate-at-50-1-snap.png
Comment (alantreadway) [reply packages/ag-charts-community/src/chart/series/cartesian/__image_snapshots__/bar-series-test-ts-bar-series-add-animation-for-column-time-x-axis-number-y-axis-should-animate-at-50-1-snap.png:1]: The add phase is in the final 25% of the alotted time, so this tallies with that.

Comment (alantreadway) [reply packages/ag-charts-community/src/chart/series/cartesian/__image_snapshots__/bar-series-test-ts-bar-series-add-animation-for-column-time-x-axis-number-y-axis-should-animate-at-50-1-snap.png:1]: The add phase is in the final 25% of the allotted time, so this tallies with that.

**Pull request merged**